examples/dataframe/query_read_data.py

```diff
@@ -0,0 +1,12 @@
+from nisystemlink.clients.dataframe import DataFrameClient
+
+client = DataFrameClient()
+
+# List a table
+client.list_tables(take=1)
+
+# Get table metadata by table id
+client.get_table_metadata("639126ad256e5501f9a3cca5")
```

Comment:
What if we grabbed the ID of the table that gets returned from `list_tables` above instead of hard-coding one for this example?